### DISCUSSION TITLE
Should YAACC include a video player?

### DISCUSSION THREAD

**Discussion by tobexyz** [tobexyz/yaacc-code#57] [Ideas]
Hello community,
I need your feedback about the video playing behavior. 
Currently YAACC does not play videos with a builtin player. YAACC is always using an app outside of YAACC for that. 
This design was chosen because video playing may lead to some complexity because you may have to handle subtitles and so on.   
I would like to know if  YAACC should include a simple video player instead of opening another app?

Best  tobexyz

**Comment by ildar** [comment]
Hi!
First off I want to thank you for the Yaacc. I feel FOSS lacking a good
UPNP app that Yaacc partially covers.
No, I don't think the embedded video player is needed. Any VLC-ish app that
is set as default should work just fine.
What's needed is
1. rework of UI and UX
2. https://sourceforge.net/p/yaacc/tickets/117/ , the old request ))

**Reply by tobexyz** [reply]
Thank you for your response. The share feature will be included in the upcoming release 3.1.0.
Maybe I am able to release it this week.
The UI redesign has also started but that's a bigger part and I am trying to split it in smaller parts

**Comment by ildar** [comment]
That's great news!! waiting for the new code to test!

**Comment by cweiske** [comment]
I personally don't need a video player, so it's fine as it is now.
My usecase is playing music from the local UPnP server (gerbera)

**Comment by Typhonling** [comment]
I don't think adding a player is a good idea, but sharing several DLNA video streams at once to a local player's playlist should be a nice feature.This will allow players that do not support adding dlna servers to play dlna videos continuously. 
It might be a little more difficult, and perhaps there isn't a consistent interface for players to add several items to their playlists from the exterior.
